content/en/blog/2025/protect-signing-secrets-for-tekton-chains-using-confidential-computing/index.md

```diff
@@ -0,0 +1,386 @@
+---
+title: Protect Signing Secrets for Tekton Chains using Confidential Computing
+linkTitle: Protect Signing Secrets for Tekton Chains using Confidential Computing
+date: 2025-04-23
+author: "Puneet Punamiya and Pradipta Banerjee, Red Hat"
+description: >
+  Integrating Tekton pipelines and chains with confidential computing
+---
+
+
+## What are Tekton Chains?
```

Comment:
NIT: "Tekton Chains" is the name of the project, so I would say "What is Tekton Chains"